pos_cashback_warning/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 - Today Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Point of Sale - Cashback Warning",
+    "version": "12.0.1.0.1",
+    "category": "Point of Sale",
+    "author": "GRAP, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/pos",
+    "license": "AGPL-3",
+    "maintainers": ["legalsylvain"],
+    "depends": [
+        "point_of_sale",
+    ],
+    "data": [
+        "views/templates.xml",
+    ],
+    "qweb": [
+        'static/src/xml/pos.xml',
+    ],
+    "images": [
+        "static/description/pos_cashback_warning.png",
```

Comment:
Question: does this `images` key do something? 👀

Reply:
AFAIK, for the apps.odoo.com, the images is used to display the image in the appstore.

See V13 doc : https://www.odoo.com/documentation/13.0/developer/reference/addons/module.html?highlight=manifest

but in master, the key seems to be removed : https://www.odoo.com/documentation/15.0/developer/reference/backend/module.html?highlight=manifest 

I'll remove it.

thanks for the review.